lms/static/js/discussions_management/views/discussions.js

```diff
@@ -16,11 +28,148 @@
+                            selected: selectedScheme === enrollmentTrack,
+                            enabled: self.isSchemeAvailable(enrollmentTrack) || selectedScheme === enrollmentTrack
+                        },
+                        {
+                            key: cohort,
+                            displayName: gettext('Cohorts'),
+                            descriptiveText: gettext('Use cohorts as the basis for dividing discussions. All learners, regardless of cohort, see the same discussion topics, but within divided topics, only members of the same cohort see and respond to each others’ posts. '), //  eslint-disable-line max-len
+                            selected: selectedScheme === cohort,
+                            enabled: self.isSchemeAvailable(cohort) || selectedScheme === cohort
+                        }
+
+                    ];
+                },
+
+                isSchemeAvailable: function(scheme) {
+                    var self = this;
+                    return self.discussionSettings.attributes.available_division_schemes.indexOf(scheme) !== -1;
+                },
+
+                showMessage: function(message, type) {
+                    var self = this,
+                        model = new NotificationModel({type: type || 'confirmation', title: message});
+                    this.removeNotification();
+                    this.notification = new NotificationView({
+                        model: model
+                    });
+                    self.$('.division-scheme-container').prepend(this.notification.$el);
+                    this.notification.render();
+                },
+
+                removeNotification: function() {
+                    if (this.notification) {
+                        this.notification.remove();
+                    }
+                },
+
+                getSelectedScheme: function() {
+                    return this.$('input[name="division-scheme"]:checked').val();
+                },
+
+                getTopicNav: function() {
+                    return this.$('.topic-division-nav');
+                },
+
+                divisionSchemeChanged: function() {
+                    var selectedScheme = this.getSelectedScheme(),
+                        topicNav = this.getTopicNav(),
+                        messageSpan = this.$('.division-scheme-message'),
+                        fieldData = {
+                            division_scheme: selectedScheme
+                        };
+
+                    this.hideTopicNav(selectedScheme, topicNav);
+                    this.saveDivisionScheme(topicNav, fieldData);
+                    this.showSelectMessage(selectedScheme, messageSpan);
```

Comment:
We can show them in the confirmation message if you would like (where we currently just say "Your changes have been saved").